okhttp/src/main/java/okhttp3/CacheControl.kt

```diff
@@ -344,7 +405,7 @@ class CacheControl private constructor(
      */
     private fun String.indexOfElement(characters: String, startIndex: Int = 0): Int {
```

Comment:
TIOLI: We could drop this (at cost of storing immutable list as a const instead of strings) and replace with

    "asb".indexOfAny("abc".toList(), startIndex = 3)

Reply:
I think I prefer what we have here. Fewer allocations!